recipes/esys-escript/meta.yaml

```diff
@@ -0,0 +1,73 @@
+
+package:
+    name: esys-escript
+    version: 5.4
+source:
+    - svn_url: https://svn.geocomp.uq.edu.au/svn/esys13/trunk
+      svn_rev: 6850
+      folder: escript
+build:
+    skip: True  # [osx]
+    skip: True  # [win]
+    number: '0'
+    rpaths:
+        - ${PREFIX}/bin
+        - ${PREFIX}/include
+        - ${PREFIX}/lib
+        - ${SP_DIR}
+requirements:
+    build:
+        - {{ compiler('cxx') }}
+        - svn
+        - scons 3.0.5 py27_0
```

Comment:
Why pin this so strictly?

Reply:
My code requires a newer version of scons in order to compile.

I have changed this to:
scons >3.0.5